CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 7.8.0 (May 4, 2023). Tested on Artifactory 7.55.10
```

Comment:
The version should be 7.7.0, since the previous PR was closed. Please fix the conflict.